sdk/src/client/bindings/python/iota_client/types/unlock_condition.py

```diff
@@ -31,8 +31,8 @@ def __init__(self, type=None, address=None, amount=None, unix_time=None, return_
         self.type = type
         self.address = address
         self.amount = amount
-        self.unix_time = unix_time
-        self.return_address = return_address
+        self.unixTime = unix_time
+        self.returnAddress = return_address
```

Comment:
You sometimes camelize the actual fields and sometimes not, like it's camelized here but not in TokenScheme

Reply:
Idk if the fields should be camelized, depending on Python convention, but you choose, I don't mind much

Reply:
For TokenScheme it wasn't needed because they got converted first, in the unlock condition not, but aligned now